src/primitives/sphere.jl

```diff
@@ -37,6 +37,36 @@ end
 
 Sphere(p1::Tuple, p2::Tuple, p3::Tuple) = Sphere(Point(p1), Point(p2), Point(p3))
 
+"""
+    Sphere(p1, p2, p3, p4)
+
+A 3D sphere passing through points `p1`, `p2`, `p3` and `p4`.
+"""
+function Sphere(p1::Point{3}, p2::Point{3}, p3::Point{3}, p4::Point{3})
+  v1 = coordinates(p1)
+  v2 = coordinates(p2)
+  v3 = coordinates(p3)
+  v4 = coordinates(p4)
+  M = hcat(v1, v2, v3, v4)
+  a = 2 * det(vcat(M, ones((1, 4))))
+  if a == 0
+    error("The four points are coplanar.")
+  end
+  q = [dot(v1, v1), dot(v2, v2), dot(v3, v3), dot(v4, v4)] / a
```

Comment:
The dot product can be written with LaTex by writting \dot and pressing tab

Reply:
I don't understand. This makes a dot above a symbol, like an accented letter with a dot for the accent.

Reply:
Try x \dot y (with whitespaces)

Reply:
Still get this strange dot.

Reply:
Sorry, my mistake, I meant \cdot for the linear algebra dot product in LaTeX

Reply:
It's `\cdot`, no?